lib/phx_tools_web/live/phx_tools_live/index.html.heex

```diff
@@ -1,9 +1,8 @@
 <PhxToolsComponents.os_landing_card
   operating_system={@operating_system}
   live_action={@live_action}
-  source_code_url={@source_code_url}
 >
   <:installation_command>
-    <%= installation_command(@live_action) %>
+    <%= installation_command() %>
```

Comment:
Maybe pass the OS, not the live_action?

Reply:
Do we really need to pass anything since the command is one?
I had removed the "@live_action".

Reply:
My bad. You were removing the `live_action` from here, not adding anything.
I'm hallucinating 😅